Author: Tianon Gravi <admwiggin@gmail.com>
Description: "archive/tar" patch for upstreamed xattrs patch
Applied-Upstream: golang-1.3 (scheduled to be released 2014-06-01)

diff --git a/archive/archive.go b/archive/archive.go
index 16c0199..817d37e 100644
--- a/archive/archive.go
+++ b/archive/archive.go
@@ -2,7 +2,7 @@ package archive
 
 import (
 	"bytes"
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"compress/bzip2"
 	"compress/gzip"
 	"errors"
diff --git a/archive/archive_test.go b/archive/archive_test.go
index 8badd58..4f6f5df 100644
--- a/archive/archive_test.go
+++ b/archive/archive_test.go
@@ -2,7 +2,7 @@ package archive
 
 import (
 	"bytes"
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"fmt"
 	"io"
 	"io/ioutil"
diff --git a/archive/changes.go b/archive/changes.go
index b46b13b..25406f5 100644
--- a/archive/changes.go
+++ b/archive/changes.go
@@ -1,7 +1,7 @@
 package archive
 
 import (
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"fmt"
 	"github.com/dotcloud/docker/utils"
 	"io"
diff --git a/archive/diff.go b/archive/diff.go
index 6a77839..7bf01f3 100644
--- a/archive/diff.go
+++ b/archive/diff.go
@@ -1,7 +1,7 @@
 package archive
 
 import (
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"fmt"
 	"io"
 	"io/ioutil"
diff --git a/archive/wrap.go b/archive/wrap.go
index 981420b..0abe320 100644
--- a/archive/wrap.go
+++ b/archive/wrap.go
@@ -2,7 +2,7 @@ package archive
 
 import (
 	"bytes"
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"io/ioutil"
 )
 
diff --git a/integration/api_test.go b/integration/api_test.go
index 5779e6b..e8ef626 100644
--- a/integration/api_test.go
+++ b/integration/api_test.go
@@ -3,7 +3,7 @@ package docker
 import (
 	"bufio"
 	"bytes"
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"encoding/json"
 	"fmt"
 	"github.com/dotcloud/docker"
diff --git a/integration/utils_test.go b/integration/utils_test.go
index 947ace1..59297fa 100644
--- a/integration/utils_test.go
+++ b/integration/utils_test.go
@@ -2,7 +2,7 @@ package docker
 
 import (
 	"bytes"
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"fmt"
 	"io"
 	"io/ioutil"
diff --git a/utils/tarsum.go b/utils/tarsum.go
index ddeecfb..be8b4db 100644
--- a/utils/tarsum.go
+++ b/utils/tarsum.go
@@ -2,7 +2,7 @@ package utils
 
 import (
 	"bytes"
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"compress/gzip"
 	"crypto/sha256"
 	"encoding/hex"
diff --git a/utils_test.go b/utils_test.go
index 6917007..415391c 100644
--- a/utils_test.go
+++ b/utils_test.go
@@ -2,7 +2,7 @@ package docker
 
 import (
 	"bytes"
-	"code.google.com/p/go/src/pkg/archive/tar"
+	"archive/tar"
 	"io"
 )
 
